src/dynamixel/controllers/usb2dynamixel.hpp

```diff
@@ -23,12 +23,14 @@ namespace dynamixel {
         class Usb2Dynamixel {
             // TODO : declare private copy constructor and assignment operator
         public:
-            Usb2Dynamixel(const std::string& name, int baudrate = B115200, double recv_timeout = 0.1) : _recv_timeout(recv_timeout), _fd(-1)
+            Usb2Dynamixel(const std::string& name, int baudrate = B115200, double recv_timeout = 0.1)
+                : _recv_timeout(recv_timeout), _fd(-1), _report_bad_packet(false)
```

Comment:
Yes, by default. The idea is that is a debug-targeted feature and I didn't want to bother the user with messages that she can ignore.
It actually is also an experiment. I'll see what it does on our robots and adapt the code if necessary.